tests/integration/questionnaire/test_questionnaire_final_confirmation.py

```diff
@@ -16,7 +16,7 @@ def test_final_confirmation_asked_at_end_of_questionnaire(self):
         post_data = {
             'action[start_questionnaire]': 'Start Questionnaire'
         }
-        resp = self.get_and_post_with_csrf_token(base_url + '14ba4707-321d-441d-8d21-b8367366e766/0/introduction', data=post_data, follow_redirects=False)
+        resp = self.get_and_post_with_csrf_token(base_url + 'final-confirmation/0/introduction', data=post_data, follow_redirects=False)
```

Comment:
Spotted it whilst fixing some of the failing tests. If I remember, it was working because the response code was still 302 so it passed the assert because the view handler was issuing a redirect even through the group was incorrect.